pkgs/by-name/em/empty-epsilon/package.nix

```diff
@@ -22,7 +22,7 @@ let
       owner = "daid";
       repo = "SeriousProton";
       rev = "EE-${version.seriousproton}";
-      hash = "sha256-byLk4ukpj+s74+3K+1wzRTXhe4pKkH0pOSYeVs94muc=";
+      hash = "sha256-B7BUe5rtN/eABJwkuSyn+h1lIHuV/tZUNcGXTyaWIr4=";
     };
 
     nativeBuildInputs = [ cmake ];
```

Comment:
line 55 should be sha256-DxaasUyJa8n0ha8RqAfEnqCVELs5Or0zvIOgcK75TnU=, right?

Reply:
No, with that hash it won't build.

Reply:
right now we are using the hash of the older version 😅